src/rdf4cpp/IRI.cpp

```diff
@@ -140,8 +140,17 @@ bool IRI::is_iri() const noexcept { return true; }
 
 
 IRI IRI::default_graph(storage::DynNodeStoragePtr node_storage) {
-    return IRI::make_unchecked("", node_storage);
+    auto const id = datatypes::registry::reserved_datatype_ids[datatypes::registry::default_graph_iri];
```

Comment:
Note to myself: this is strange because of https://github.com/rdf4cpp/rdf4cpp/issues/243